src/brogue/Rogue.h

```diff
@@ -1365,6 +1365,7 @@ typedef struct item {
     short yLoc;
     keyLocationProfile keyLoc[KEY_ID_MAXIMUM];
     short originDepth;
+    unsigned long lastUsed;         // Player turn last applied
```

Comment:
I deliberately called this `lastUsed` rather than `lastApplied`, in case we ever choose to overload it for (say) armor or weapons, where the relevant action would probably be equip rather than apply.